demos/imagenet_classification_demo/main.cpp

```diff
@@ -0,0 +1,511 @@
+        
+        // -------------------------------------Processing infer requests-------------------------------------
+        do {
+            //std::cout << "Num completed: " <<  completedInferRequests.size() << std::endl;
+            if (isTestMode && elapsedSeconds >= std::chrono::seconds{5}) {
+                isTestMode = false;
+                gridMat = GridMat(presenter, cv::Size(width, height), cv::Size(16, 9), avgFPS);
+                startTickCount = std::chrono::steady_clock::now();
+                framesNum = 0;
+                latencySum = 0;
+            }
+
+            std::unique_lock<std::mutex> lock(mutex);
+            if (!completedInferRequests.empty()) {
+                auto completedInferRequestInfo = completedInferRequests.front();
+                completedInferRequests.pop();
+                lock.unlock();
+                emptyInferRequests.push({completedInferRequestInfo.inferRequest, 
+                                         std::vector<InferRequestInfo::InferRequestImage>()});
+
+                std::vector<unsigned> rightClasses = {};
+                for (size_t i = 0; i < completedInferRequestInfo.images.size(); i++) {
+                    rightClasses.push_back(completedInferRequestInfo.images[i].rightClass);
+                }
+
+                std::vector<std::vector<unsigned>> results = topResults(
+                    *completedInferRequestInfo.inferRequest.GetBlob(outputName), FLAGS_nt);
+                std::vector<std::string> predictedLabels = {};
+                for (size_t i = 0; i < FLAGS_b; i++) {
+                    int predictionResult = -1;
+                    if (!FLAGS_gt.empty()) {
+                        for (size_t j = 0; j < FLAGS_nt; j++) {
+                            unsigned predictedClass = results[i][j];
+                            if (predictedClass == rightClasses[i]) {
+                                predictionResult = 1;
+                                predictedLabels.push_back(labels[predictedClass]);
+                                correctPredictionsCount++;
+                                break;
+                            }
+                        }
+                    } else {
+                        predictionResult = 0;
+                    }
+
+                    if (predictionResult != 1) {
+                        predictedLabels.push_back(labels[results[i][0]]);
+                    }
+                    totalPredictionsCount++;
+
+                    shownImagesInfo.push_back(
+                        std::make_tuple(completedInferRequestInfo.images[i].mat, predictedLabels[i], predictionResult));
+                }
+
+                framesNum += FLAGS_b;
+
+                avgFPS = 1. * framesNum / std::chrono::duration_cast<std::chrono::milliseconds>(
+                    std::chrono::steady_clock::now() - startTickCount).count() * 1000;
```

Comment:
`typedef std::chrono::duration<double, std::chrono::seconds::period> Sec;`
and casting `std::chrono::steady_clock::now() - startTickCount` to `Sec` will eliminate that `* 1000`

Reply:
`1. * ` is no longer needed.